tests/unit/test_connection_cmds.py

```diff
@@ -1227,7 +1222,7 @@ class TestSubcmdClass(CLITestsBase):
         "desc, inputs, exp_response, mock, condition",
         TEST_CASES)
     def test_connection(self, desc, inputs, exp_response, mock, condition,
-                        default_connections_file_path):
+                        default_connections_file_path, verbose=True):
```

Comment:
The `verbose` argument causes this Pylint issue:
```
************* Module tests.unit.test_connection_cmds
tests/unit/test_connection_cmds.py:1225:55: W0613: Unused argument 'verbose' (unused-argument)
```

Reply:
Whoops forgot to remove that one after a test.

Reply:
DONE